bodhi/server/models.py

```diff
@@ -1832,6 +1832,8 @@ def check_karma_thresholds(self, db, agent):
         if self.autokarma and self._composite_karma[1] != 0:
             log.info("Disabling Auto Push since the update has received negative karma")
             self.autokarma = False
+            text = unicode(config.get('disable_automatic_push_to_stable'))
```

Comment:
It would be a good idea to put a default value here, so that Bodhi won't crash if this new config value isn't added to the config. I like to follow [Semantic Versioning](https://semver.org) where possible, so this would be an easy way to avoid a backwards incompatible change.

I think this might be as simple as changing the above line to something like this:

```
default_comment = u'Bodhi is disabling automatic push to stable due to negative karma. The maintainer may manually if they determine that the issue is not severe.'
text = unicode(config.get('disable_automatic_push_to_stable', default_comment))
```